DOCKER_HUB.md

```diff
@@ -0,0 +1,52 @@
+# Quick reference
+
+-    **Maintained by**:  
+      [georchestra.org](https://www.georchestra.org/)
+
+-    **Where to get help**:  
+     the [Georchestra Github repo](https://github.com/georchestra/georchestra), [Matrix chat](https://matrix.to/#/#georchestra:libera.chat), Stack Overflow
```

Comment:
Matrix chat is not the native (official ?) chat. 
Maybe we can insert https://kiwiirc.com/nextclient/irc.libera.chat/georchestra link ?

Reply:
I think we can link both matrix and IRC. Both are interconnected between each other.

Reply:
> Both are interconnected between each other.

I guess this is not transparent for everyone. Newcomers have to know that both exists and I suggest to be consistent with georchestra web site (at least show same informations - IRC)

https://www.georchestra.org/fr/communaute.html

![image](https://github.com/georchestra/mapstore2-georchestra/assets/16317988/15c48551-c1c3-404e-a9dd-7e3d08ce7664)



**Clearly, this is a detail.**

Reply:
ok maybe we can only link IRC for the moment but latter on propose matrix too on official georchestra website then changing the docker hub description.